ptbcontrib/decorators/README.md

```diff
@@ -0,0 +1,35 @@
+## Register a function as a handler (decorator)
```

Comment:
I addition to a code example, please also add some descriptive text explaining what this contribution does, what it's intended for, and possibly also other things the user should know about